wheatley/main.py

```diff
@@ -28,9 +28,9 @@
 
 def create_row_generator(args: argparse.Namespace) -> RowGenerator:
     """ Generates a row generator according to the given CLI arguments. """
-    if "comp_url" in args and args.comp_url is not None:
```

Comment:
I think this argument is in 0.5.0? Just checking you're happy to just rename it (rather than have both versions for a release, I guess it's rarely used). And should the rename should be in the change log?

Reply:
That's true - I don't think anyone knows it exists.  I'll rename it in the CHANGE_LOG.